Makefile

```diff
@@ -1,193 +1,43 @@
-GRYPE_PATH ?= ../grype
-GRYPE_DB_PATH ?= ../grype-db
-
-# Command templates #################################
-
-CRANE = $(TEMP_DIR)/crane
-CHRONICLE = $(TEMP_DIR)/chronicle
-GLOW = $(TEMP_DIR)/glow
-PUBLISH_CMD = poetry publish --build -n
-
-# Tool versions #################################
-CHRONICLE_VERSION = v0.8.0
-GLOW_VERSION = v1.4.1
-CRANE_VERSION = v0.16.1
-
-# Formatting variables #################################
-BOLD := $(shell tput -T linux bold)
-PURPLE := $(shell tput -T linux setaf 5)
-GREEN := $(shell tput -T linux setaf 2)
-CYAN := $(shell tput -T linux setaf 6)
-RED := $(shell tput -T linux setaf 1)
-RESET := $(shell tput -T linux sgr0)
-TITLE := $(BOLD)$(PURPLE)
-SUCCESS := $(BOLD)$(GREEN)
-ERROR := $(BOLD)$(RED)
-
-# this is the python package version for vunnel, based off of the git state
-# note: this should always have a prefixed "v"
-PACKAGE_VERSION = v$(shell poetry run dunamai from git --style semver --dirty --no-metadata)
-COMMIT = $(shell git rev-parse HEAD)
-COMMIT_TAG = git-$(COMMIT)
-
-
-ifndef PACKAGE_VERSION
-	$(error PACKAGE_VERSION is not set)
-endif
-
-.DEFAULT_GOAL := all
-
-.PHONY: all
-all: static-analysis test  ## Run all validations
-
-.PHONY: static-analysis
-static-analysis: virtual-env-check  ## Run all static analyses
-	pre-commit run -a --hook-stage push
-
-.PHONY: test
-test: unit  ## Run all tests
-
-virtual-env-check:
-	@ if [ "${VIRTUAL_ENV}" = "" ]; then \
-		echo "$(ERROR)Not in a virtual environment. Try running with 'poetry run' or enter a 'poetry shell' session.$(RESET)"; \
-		exit 1; \
-	fi
+TOOL_DIR = .tool
+BINNY = $(TOOL_DIR)/binny
+TASK = $(TOOL_DIR)/task
 
+.DEFAULT_GOAL := make-default
 
 ## Bootstrapping targets #################################
 
-.PHONY: bootstrap
-bootstrap: $(TEMP_DIR)  ## Download and install all tooling dependencies
-	curl -sSfL https://raw.githubusercontent.com/anchore/chronicle/main/install.sh | sh -s -- -b $(TEMP_DIR)/ $(CHRONICLE_VERSION)
-	GOBIN="$(abspath $(TEMP_DIR))" go install github.com/charmbracelet/glow@$(GLOW_VERSION)
-	GOBIN="$(abspath $(TEMP_DIR))" go install github.com/google/go-containerregistry/cmd/crane@$(CRANE_VERSION)
-
-$(TEMP_DIR):
-	mkdir -p $(TEMP_DIR)
-
-$(BIN_DIR):
-	mkdir -p $(BIN_DIR)
-
-## Development targets #################################
-
-.PHONY: dev
-dev:  ## Get a development shell with locally editable grype, grype-db, and vunnel repos
-	@DEV_VUNNEL_BIN_DIR=$(ABS_BIN_DIR) .github/scripts/dev-shell.sh $(provider) $(providers)
-
-.PHONY: build-grype
-build-grype: $(BIN_DIR) ## Build grype for local development
-	@cd $(GRYPE_PATH) && go build -o $(ABS_BIN_DIR)/grype ./cmd/grype
-
-.PHONY: build-grype-db
-build-grype-db: $(BIN_DIR) ## Build grype-db for local development
-	@cd $(GRYPE_DB_PATH) && go build -o $(ABS_BIN_DIR)/grype-db ./cmd/grype-db
-
-.PHONY: update-db
-update-db: check-dev-shell ## Build and import a grype database based off of the current configuration
```

Comment:
It looks like `update-db` also didn't make it to the task file. Is this intentional?

Reply:
I missed all of the dev targets, I'll push an update